static/src/stylesheets/layout/_ab-new-header-test-variant.scss

```diff
@@ -750,8 +750,8 @@ $navigation-horizontal-padding: $gs-gutter * 2 + $gs-gutter/2;
 }
 
 .tertiary-navigation__link {
-    position: relative;
     @include fs-textSans(4);
+    position: relative;
     color: $neutral-4;
     display: block;
     line-height: 1.5em;
```

Comment:
Do you mean the em values in this stuff? If so it's because the font-size is in vw, so want the sizing to be proportionate.